app/util/autoversioning.py

```diff
@@ -170,3 +175,12 @@ def _execute_local_git_command(*args):
         cwd=os.path.dirname(__file__),
     )
     return command_output.decode()
+
+
+if __name__ == '__main__':
+    """
+    Print version string.
+    """
+    version = get_version()
+    write_package_version_file(version)
+    print(version)
```

Comment:
The `rpmbuild` needs to include the file in package. So I think we can not remove the file in the `autoprovisioning` script. I think we need to do it from Makefile.
Let me know if I am missing something.